src/ws/service/friendships_service.rs

```diff
@@ -67,25 +71,30 @@ impl FriendshipsServiceServer<SocialContext, RPCFriendshipsServiceError> for MyF
         request: Payload,
         context: ProcedureContext<SocialContext>,
     ) -> Result<ServerStreamResponse<UsersResponse>, RPCFriendshipsServiceError> {
+        let start = Instant::now();
+        let metrics = context.server_context.metrics.clone();
+        record_procedure_call_size(metrics.clone(), Procedure::GetFriends, &request).await;
```

Comment:
shouldn't we sum the request and response size to report the metric? this way we are measuring only the request size, is that okay?

Reply:
oh, yes. we're only measuring only the request, but because I thought that the idea of the metric is to provide insight into how clients are using the service. I don't have much experience defining these kind of metrics, so I don't know if it's okay haha. I can change it to report the sum of both if it's more appropriate 🏃🏻‍♀️